example/lib/popup_outside_of_map.dart

```diff
@@ -10,10 +10,10 @@ class PopupOutsideOfMap extends StatefulWidget {
   const PopupOutsideOfMap({Key? key}) : super(key: key);
 
   @override
-  State<PopupOutsideOfMap> createState() => _PopupOutsideOfMapState();
+  State<PopupOutsideOfMap> createState() => _PopupOutsideOfFlutterMapState();
 }
 
-class _PopupOutsideOfMapState extends State<PopupOutsideOfMap> {
+class _PopupOutsideOfFlutterMapState extends State<PopupOutsideOfMap> {
```

Comment:
I think this renaming is a search-replace error. _PopupOutsideOfMapState means the state of PopupOutsideOfMap.

Reply:
Haha I think you are right! Will fix!